Bottom: 78b4dd5127319a6da527cee309e02a2161ff148e
Top:    9c346b48a16c31317a1cbf6c48fc270d0aa20c03
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-12 03:28:31 +0100

Refresh of off-to-find-burried-treasure-o

---

diff --git a/src/pour.c b/src/pour.c
index e7d8920..ac1603a 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -671,7 +671,14 @@ mark_islands (DataTypePtr Data, LayerTypePtr layer, PourTypePtr pour,
               int type, void *ptr1, void *ptr2)
 {
   struct cpInfo info;
-  info.region = &((PinTypePtr) ptr2)->BoundingBox;
+  BoxType region = ((PinTypePtr) ptr2)->BoundingBox;
+
+  region.X1 -= UNSUBTRACT_BLOAT;
+  region.Y1 -= UNSUBTRACT_BLOAT;
+  region.X2 += UNSUBTRACT_BLOAT;
+  region.Y2 += UNSUBTRACT_BLOAT;
+
+  info.region = &region;
   info.layer = layer;
 
   return r_search (pour->polygon_tree, info.region, NULL,
